scripts/ibmcloud_post_install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -e
+
+### Status flag
+
+log () {
+  FLAGFILE=/root/ibmcloud-post-install.log
+  echo "$(date) +++ $1" >> $FLAGFILE
+}
+
+log "Starting ibmcloud-post-install.sh"
+
+### SSH hardening
+### IBM Cloud machines by default allow SSH as root using the password. Given that the default one
+### is extremely simple, we disable this option.
+
+sed -i "s/.*RSAAuthentication.*/RSAAuthentication yes/g" /etc/ssh/sshd_config
+sed -i "s/.*PubkeyAuthentication.*/PubkeyAuthentication yes/g" /etc/ssh/sshd_config
+sed -i "s/.*PasswordAuthentication.*/PasswordAuthentication no/g" /etc/ssh/sshd_config
+
+systemctl restart sshd.service
+
+log "SSH configuration applied successfuly"
+
+### SSH local key
+
+mkdir -p /root/.ssh
+chmod 700 /root/.ssh
+ssh-keygen -t rsa -b 4096 -N "" -f /root/.ssh/id_rsa
+cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys
```

Comment:
What do we need to authorize the local root key for?

Reply:
I often find myself doing `ssh localhost`